compiler/src/iree/compiler/Codegen/Common/test/linearize_memrefs.mlir

```diff
@@ -0,0 +1,65 @@
+// RUN: iree-opt -iree-linearize-memrefs -allow-unregistered-dialect %s | FileCheck %s
```

Comment:
Could you also add some tests with dynamic shapes, and where the strides and offsets are also dynamic?